### DISCUSSION TITLE
cannot connect to some (two) of my many Windows-clients

### DISCUSSION THREAD

**Discussion by paulsommer** [rustdesk/rustdesk#9911] [Q&A]
I am using rustdesk for a long time with a self hosted connection server and a Linux PC on my side and mostly Windows Systems on the other side.
There are two Windows Systems to where I cannot establish a connection

Upon connection Rustdesk says: "Connection is being established" (Verbindung wird hergestellt) And that's it. The Client doesn't see the dialog to accept the connection.

My ticket was closed immediately with the note that it was a build problem. I can't understand that.

What can I do to narrow down the problem?

**Comment by Poordeveloper** [comment]
Go to RustDesk discord, you won't get any help here I believe. They offer you free tool, it does not mean they will offer you free support.
